src/components/CommentForm.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+import { useContext } from "react";
+import { UserContext } from "../context/userContext";
+import { postArticleComment } from "../api";
+
+export default function CommentForm({ article_id }) {
+  const { user } = useContext(UserContext);
+  const [comment, setComment] = useState("");
+  const [isError, setIsError] = useState(null);
+  const [isPosted, setIsPosted] = useState(null);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!comment.trim()) {
+      setIsError("Say something!");
+      return;
+    }
+    postArticleComment(article_id, user, comment)
+      .then(() => {
+        setComment("");
+        setIsPosted("Comment posted!");
```

Comment:
Do you wanted this UI to persist? You could use a setTimeout to clear the state after some seconds.